advocacy_docs/partner_docs/CommVaultGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Commvault'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Commvault       |
+| **Partner Product**   |  Commvault Backup & Recovery       |
+| **Web Site**      |    https://www.commvault.com/   |
+| **Version**   |  Commvault Backup & Recovery 11.24: Microsoft Windows Server 2019 Standard      |
+| **Product Description**      |  Wherever your data resides, ensure availability via a single interface with Commvault Backup & Recovery. Say goodbye to costly data loss scenarios, segregated data silos, missing recovery SLAs and inefficient scaling. Empower your team to do more.     |
```

Comment:
change description to
Wherever your data resides, ensure availability via a single interface with Commvault Backup & Recovery.
Comprehensive workload coverage, files, apps, and databases including EDB Postgres Advanced Server and EDB Postgres Extended Server from a single extensible platform and user interface.
CommVault Backup & Recovery provides a comprehensive backup and archiving solution for your trusted recovery, ransomware protection, and security.